### PR TITLE
GreasedLineTools.GetPointsFromText

### PR DESCRIPTION
**Pull request by RolandCsibrei**
Gets 3D positions of points from a text and font

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/13989/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13989/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13989/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/13989/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/13989/merge
https://gui.babylonjs.com/?snapshot=refs/pull/13989/merge
https://nme.babylonjs.com/?snapshot=refs/pull/13989/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/13989/merge#BCU1XR#0

File: packages/dev/core/src/Meshes/Builders/textBuilder.ts [hunk 1]
```diff
@@ -183,7 +183,15 @@ function CreateShapePath(
...
-function CreateShapePaths(text: string, size: number, resolution: number, fontData: IFontData) {
+/**
+ * Creates shape paths from a text and font
+ * @param text the text
+ * @param size size of the font
+ * @param resolution resolution of the font
+ * @param fontData defines the font data (can be generated with http://gero3.github.io/facetype.js/)
+ * @returns array of ShapePath objects
+ */
+export function CreateTextShapePaths(text: string, size: number, resolution: number, fontData: IFontData) {
```
Comment (Popov72) [inline packages/dev/core/src/Meshes/Builders/textBuilder.ts:194]: The function should not be renamed to avoid breaking changes.

**Review by Popov72 — CHANGES_REQUESTED** [review]

File: packages/dev/core/src/Meshes/Builders/textBuilder.ts (hunk 1 above)
Comment (RolandCsibrei) [reply packages/dev/core/src/Meshes/Builders/textBuilder.ts:194]: Yes I know, but this function was private and was used only once in the same source file ;-)

Comment (Popov72) [reply packages/dev/core/src/Meshes/Builders/textBuilder.ts:194]: Ah yes, I did not see the function was not exported previously!

All good then.

**Review by Popov72 — APPROVED** [review]

**Pull request merged**